File: Backend/blocked-users/api/delete/service.php
<?php
require_once '../../db/database.php';

function deleteBlockedUser($id) {
    $db = new Database();
    $stmt = $db->conn->prepare("DELETE FROM blocked_users WHERE id = ?");
    
    try {
        $stmt->execute([$id]);
        return ["message" => "User deleted successfully"];
    } catch (PDOException $e) {
        return ["error" => "Error deleting user"];
    }
}
?>

File: Backend/blocked-users/api/delete/service.rb
require_relative "../../db/database"

class DeleteBlockedUserService
  def self.delete(email)
    db = Database.connection
    db.execute("DELETE FROM blocked_users WHERE email = ?", [email])
    { message: "User unblocked successfully" }
  rescue SQLite3::Exception => e
    { error: "Error deleting user: #{e.message}" }
  end
end
